pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -7,11 +7,11 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "soundconverter";
-  version = "4.0.3";
+  version = "4.0.4";
 
   src = fetchurl {
     url = "https://launchpad.net/soundconverter/trunk/${version}/+download/${pname}-${version}.tar.gz";
-    sha256 = "sha256-hzIG/4LD3705erPYvXb7uoRwF9LtKKIKB3jrhpYMsZ0=";
+    sha256 = "sha256-15TeNxFQzE6pzADuBN8jEzpPPna4pt0x4GXsaxi0gr4=";
```

Comment:
(non-blocking)
```suggestion
    hash = "sha256-15TeNxFQzE6pzADuBN8jEzpPPna4pt0x4GXsaxi0gr4=";
```

Reply:
Oh, the argument name changed? Didn't know.